StreamVideoTests/IntegrationTests/CallCRUDTests.swift

```diff
@@ -355,7 +355,7 @@ class CallCRUDTest: IntegrationTest {
         XCTAssertEqual(participants.last?.hasAudio, true, "Participant should have audio enabled")
 
         try await firstUserCall.muteAllUsers()
-        try await customWait(nanoseconds: 15_000_000_000)
+        try await customWait(nanoseconds: 18_000_000_000)
```

Comment:
That grows up very quickly. We will soon need to identify the problem here or improve those tests

Reply:
I would actually disable this test and fix it separately @testableapple